projectData.js

```diff
@@ -0,0 +1,38 @@
+const projectData = [
+  {
+    name: 'Winter',
+    palettes: [
+      {  
+        name: 'Christmas Theme',
+        color1: '#800000',
```

Comment:
We may not want to save "#" symbol in the database, however we can look into this when we build the front end.